app/views/compute_resources_vms/show/_libvirt.html.erb

```diff
@@ -40,7 +40,7 @@
           <td><%= "#{nic.bridge} - #{nic.mac} (#{nic.model})" %></td>
         </tr>
       <% end %>
-      <% @vm.volumes.each do |vol| %>
+      <% @vm.volumes.compact.each do |vol| %>
```

Comment:
I wonder why it's returning `nil` in that list. Feels like an upstream regression as well. Could you have a look if we can solve it in fog-libvirt?